storage/time-series/influxdb/retriever/app/express/index.js

```diff
@@ -73,17 +73,15 @@ module.exports = (routes, serviceState, openApiPath) => {
         stateManager: serviceState,
         logger,
       }),
-      responseCompressInterceptor({
-        supportTrustProxy: configExpress.trustproxy,
-      }),
+      responseCompressInterceptor({}),
       requestLogInterceptor({
         logger,
       }),
     ],
     routes: (routes).flat(),
     logger,
     config: {
-      trustproxy: configExpress.trustproxy,
+      supportTrustProxy: configExpress.trustproxy,
     },
```

Comment:
The `config` object no longer exists, pass the attribute `supportTrustProxy` directly

https://github.com/dojot/dojot-microservice-sdk-js/blob/master/lib/webUtils/framework/createExpress.js#L88